launcher/.gitignore

```diff
@@ -0,0 +1,2 @@
+cover.out
+src/main/resources/binaries/**/launcher
```

Comment:
Either the binaries should be checked in, or they should be generated in the targets folder. 

It's bad form to place generated files in the source folder. For example, commands such as "mvn clean" don't work properly since they don't understand that those are build artifacts, not source artifacts.

Reply:
Moved to target